webdriver-spec.html

```diff
@@ -2995,12 +2995,19 @@
  run the following steps:
 
 <ol>
+ <li><p>Start a <var>timer</var>. If this algorithm has not
+  completed before <var>timer</var> reaches
+  the <a>session</a>'s <a>session page load timeout</a> in
+  milliseconds, return an <a>error</a> with <a>error
+  code</a> <a>timeout</a>.
+
  <li><p>If the <a>current session</a>'s <a>page loading strategy</a>
-  is not <code>"none"</code> and if there is an ongoing attempt to
+  is not "<code>none</code>" and if there is an ongoing attempt to
   <a>navigate</a> the <a>current browsing context</a> that has not
-  yet <a>matured</a>, wait for navigation to complete.
+  yet <a>matured</a>, wait for navigation to <a>mature</a>.
```

Comment:
Cancelling it seems unwise, since the navigation could have been started by a click issued by webdriver --- we use this algorithm in the `Processing Model`, `Element Click` as well as `Go` and `Refresh`.